bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Triboson_ZPrimeToZRToZWW/Triboson_M_R/Triboson_M_R_customizecards.dat

```diff
@@ -0,0 +1,6 @@
+set param_card kkinputs 5 <MASS>
+set param_card kkinputs 2 6.708000e+00
+set param_card mass 9000025 <MASS_R>
+set param_card mass 9000023 <MASS>
+set param_card DECAY 9000023 1.195600e+01
+set param_card DECAY 9000025 0.31035
```

Comment:
Do you consider the width stable across the mass range you are probing? The values seems very specific.

Reply:
We use different width for different signal point. I upload file "width.txt" which contains widths of zkk and r for different signal points and they are calculated in the way as the author of the model recommended.